.vscode/launch.json

```diff
@@ -0,0 +1,10 @@
+{
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "name": "http://127.0.0.1:5500/v1.1.1/",
+            "request": "launch",
+            "url": "http://127.0.0.1:5500/v1.1.1/"
+        }
+    ]
+}
```

Comment:
Where did this file/folder come from? I definitely don't think it's needed for using the LiveServer extension. Was it from trying to run it through the usual VSCode Run menu? Either way, I'd rather not merge this file in since it definitely isn't necessary.